### PR TITLE
dynamically including api's in python's package init

### PR DESCRIPTION
**Pull request by toddbellMSFT**

### CODE REVIEW

File: SDKGenerator.njsproj [hunk 1]
```diff
@@ -506,7 +506,7 @@
     <Content Include="targets\PythonSdk\source\playfab\PlayFabHTTP.py" />
     <Content Include="targets\PythonSdk\source\playfab\PlayFabManualTest.py" />
     <Content Include="targets\PythonSdk\source\playfab\PlayFabSettings.py" />
-    <Content Include="targets\PythonSdk\source\playfab\__init__.py" />
+    <Content Include="targets\PythonSdk\templates\__init__.py.ejs" />
```
Comment (pgilmorepf) [inline SDKGenerator.njsproj]: You don't need to move this into the templates folder.
You can just rename it in-place to append ".ejs" on the end, and it automatically gets templatized where it is.

File: targets/PythonSdk/make.js [hunk 2]
```diff
@@ -30,6 +30,10 @@ exports.makeCombinedAPI = function (apis, sourceDir, apiOutputDir) {
     console.log("Generating Combined Client/Server api from: " + sourceDir + " to: " + apiOutputDir);
 
     templatizeTree(locals, path.resolve(sourceDir, "source"), apiOutputDir);
+
+    var iniTemplate = getCompiledTemplate(path.resolve(sourceDir, "templates/__init__.py.ejs"));
```
Comment (pgilmorepf) [inline targets/PythonSdk/make.js]: This is not necessary if you leave it in source/playfab and let templatizeTree do the work.

File: targets/PythonSdk/templates/__init__.py.ejs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<% for (var a = 0; a < apis.length; a++) { var api = apis[a];
```
Comment (pgilmorepf) [inline targets/PythonSdk/templates/__init__.py.ejs]: This file looks good except the part where it's been moved.

File: targets/PythonSdk/make.js [hunk 4]
```diff
@@ -30,6 +30,7 @@ exports.makeCombinedAPI = function (apis, sourceDir, apiOutputDir) {
     console.log("Generating Combined Client/Server api from: " + sourceDir + " to: " + apiOutputDir);
 
     templatizeTree(locals, path.resolve(sourceDir, "source"), apiOutputDir);
+
```
Comment (toddbellMSFT) [inline targets/PythonSdk/make.js]: I'll get rid of the extra whitespace.

**Pull request merged**